aperturedb/ParallelLoader.py

```diff
@@ -106,7 +106,7 @@ def print_stats(self):
         else:
             mean = np.mean(times)
             std  = np.std(times)
-            tp = 1 / mean * self.numthreads
+            tp = 0 if mean == 0 else 1 / mean * self.numthreads
```

Comment:
does this address NaN errors on the [stats](https://github.com/aperture-data/aperturedb-python/issues/244) ?

Reply:
no, that is due to the {i_tp if self.error_counter == 0 else 'NaN'} .. but with the testing I'm adding to check for errors loading images with ImageDataCSV, we can resolve that too.